.github/workflows/sanity-check.yml

```diff
@@ -16,7 +16,7 @@ jobs:
         env:
           pull_request_number: ${{ github.event.pull_request.number }}
         run: |
-          curl --silent -X "GET" https://api.github.com/repos/debezium/debezium/pulls/$pull_request_number/commits | jq '.[] | {message: .commit.message}' | jq -r '.message' >> COMMIT_MESSAGE.txt
+          curl --silent -X "GET" https://api.github.com/repos/debezium/debezium/pulls/$pull_request_number/commits | jq '.[] | {message: .commit.message}' | jq '.message' | cut -c 2- >> COMMIT_MESSAGE.txt
```

Comment:
I think this can be simplified a bit, also without using any disk space and returning early once an invalid commit message is found:

```
COMMIT_MSGS=$(curl --silent -X "GET" https://api.github.com/repos/debezium/debezium/pulls/$pull_request_number/commits | jq '.[] | {message: .commit.message}' | jq '.message' | cut -c 2-)
while IFS= read -r line ; do
  line=${line%\"}
  echo "-> checking: $line"
  if [[ ! $line =~ (^DBZ-[[:digit:]]+)|(\[release\])|(\[jenkins-jobs\])|(\[docs\]) ]]; then
    echo "::set-output name=PREFIX::false"
    break
  fi
done <<< "$COMMIT_MSGS"
```

WDYT?

Reply:
This could even be optimized in a way that it processes all lines and report the failing commit messages at the end and then returning the `::set-output?` line at the end

Reply:
@rk3rn3r Applied the suggestions. Does it look better?